addons/crm_mail_plugin/__manifest__.py

```diff
@@ -17,6 +17,9 @@
         'views/crm_mail_plugin_lead.xml',
         'views/crm_lead_views.xml'
     ],
+    'web.assets_backend': [
+        'crm_mail_plugin/static/src/to_translate/**/*',
+    ],
```

Comment:
@nounoubensebia @tde-banana-odoo could you explain why this weird stuff was added to the manifest? :p
I think that was unnecessary and useless for the translations to work.

Cfr the new test for unknown manifest keys in #79977

Reply:
To add to odo's comment, I couldn't find anywhere where the root `web.assets_backend` key was used. I was confident it was useless until I read this branch, it looks like you try to fix something so now I'm not sure if it is actually used or not...

Reply:
I think the idea was to load the translations, so they are available in transiflex, I'm not 100% sure it is necessary

Reply:
@odony This weird useless stuff is linked to a commit explaining why we did it actually
```
In mail_plugin module, add a method which returns translatable modules,
this method will be overridden in the other modules to add their module names.

Translations are prepared via the _prepare_translations method which uses
the get_translations_for_webclient method, this way we can easily fetch
translations without having to write python code.

We also add xml files in "static/", these files contain terms to be translated,
for each plugin we create a separate xml file so that we can easily update each
plugin separately.

This implementation will allow having translations handled by Odoo, which has
several advantages:
- existing system, nothing to develop
- it will use transifex and the terms will be translated by the community
- forces that mail_client implementations to have the same logic (consistency)
- compared to other solutions which rely on python code to return translations,
  this solution is more robust as it avoids having to type the string to
  translate twice
```

It seemed to work at merge time, not sure why if the key is unused (did not check in details to be honest). If you remove it just be sure strings are still correctly extracted to be available for translations, as they are not defined in odoo source code, just defined to be sent to outlook or gmail plugins.

Cheers !

Reply:
> If you remove it just be sure strings are still correctly extracted to be available for translations

The extraction should be fine as the [extraction code](https://github.com/odoo/odoo/blob/8b468e1f7a3dfeb2bcfa83bff55c4b5adf6b2212/odoo/tools/translate.py#L1138-L1147) just check the content of the `static/src` folder, whatever the manifest says

Reply:
Cool, thanks :)

Reply:
yeah, we looked at the commit message, and nothing explains why the manifest keys had been introduced, as the translation extractor doesn't need that. So if you don't have any additional info, let's say if was some cosmic ray effect, and @Julien00859 can drop them in #79977 (rather than blessing them as "official manifest keys") ;-)

Reply:
This is probably because not a lot of people know extraction code was doing it automatically, so can be removed safely indeed.